include/vinecopulib/vinecop/implementation/fit_controls.ipp

```diff
@@ -155,11 +155,7 @@ FitControlsVinecop::get_trunc_lvl() const
 inline void
 FitControlsVinecop::set_trunc_lvl(size_t trunc_lvl)
 {
-  if (trunc_lvl == 0) { // select level (likely passed from R)
```

Comment:
Why do you remove that?

Reply:
Oh right that was a bit unrelated, but appears to be a relict from old times. I doesn't seem we're ever passing 0 from R when we mean Inf (we probably did in an earlier version though). However, nowadays we have proper support for 0-truncated vines if I remember correctly (which we didn't in the beginning). I changed this during debugging and then forgot about it.